main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -90,6 +91,24 @@ object Main {
       xstrictmono = cmdOpts.xstrictmono
     )
 
+    // configure Flix and add the paths.
```

Comment:
Why are all these lines of code moved here? I would keep them were they were.

Reply:
If the flix compiler is configured after the Packager is used then the implicit `Flix` instance will not be passed to the Packager

Reply:
I see, but now the packager sometimes creates its own Flix object and other times it doesn't. I think that is bad.

I would not pass any Flix object from main. Instead I would create a new private `newFlix` inside `Packager` and configure that correctly.

Reply:
Roger